### PR TITLE
COMMERCE-9052 Remove recalculatePrice from updateCommerceShippingMethod

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9052

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/helper/DefaultCommerceCheckoutStepHttpHelper.java [hunk 1]
```diff
@@ -594,6 +594,9 @@ private CommerceOrder _updateCommerceOrder(
 			commerceShippingOptionKey, commerceContext,
 			_portal.getLocale(httpServletRequest));
 
+		commerceOrder = _commerceOrderLocalService.recalculatePrice(
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-checkout-web/src/main/java/com/liferay/commerce/checkout/web/internal/helper/DefaultCommerceCheckoutStepHttpHelper.java:597]: This should be inside a transaction with _commerceOrderLocalService.updateCommerceShippingMethod....
See ShippingMethodCommerceCheckoutStep.java at line 283

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
See comments pls

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**